snuba/datasets/errors_replacer.py

```diff
@@ -222,12 +236,26 @@ def get_projects_query_flags(
 
     results = p.execute()
 
+    exclude_groups_keys_replacement_types = [
+        get_project_exclude_groups_replacement_type_key(exclude_groups_key)
+        for exclude_groups_key in exclude_groups_keys
+    ]
+
+    for exclude_groups_key_replacement_type in exclude_groups_keys_replacement_types:
+        p.get(exclude_groups_key_replacement_type)
```

Comment:
Can you use `mget` here?

https://redis.io/commands/mget

Reply:
I sure can!